ingestion.py

```diff
@@ -0,0 +1,329 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy.io as sio
+import os
+from datetime import datetime
+import re
+
+############## Dataset #################
+
+datadir = 'C://Users//thinh//Documents//TN-Vathes//NWB_Janelia_datasets//crcns_ssc5_data_HiresGutnisky2015//'
+metadatadir = datadir + 'metadata//'
+sessdatadir = datadir + 'datafiles//'
+
+# Set up 
+_datetimeformat1 = '%y%m%d'
+_datetimeformat2 = '%y%d%m'
+
+#################################
+import datajoint as dj
+dj.config['database.host'] = 'tutorial-db.datajoint.io'
+
+import pipeline
+from pipeline import reference
```

Comment:
rather than importing them all separately, simply import them together as in:
```suggestion
from pipeline import (reference, subject, acqusition, behavior, ephys, action)
```